internal/datastore/prepared_statements.go

```diff
@@ -59,7 +60,30 @@ const getPostsByDiscussionIDString = `
 		FROM posts p
 		INNER JOIN post_contents pc
 		ON p.post_content_id = pc.id
-		WHERE p.discussion_id = $1;`
+		WHERE p.discussion_id = $1
+		;`
+
+const getPostsByDiscussionIDFromCursorString = `
```

Comment:
This is allows for unidirectional page traversal. Bidirectional traversal would imply a separated SQL statement, as the ">" and "<" symbols are not settable as parameters.

Reply:
Makes sense -- and should be relatively straight forward to do when we need it. We don't need it now though so this is good work.